packages/clerk-js/src/ui/components/UserProfile/DeletePage.tsx

```diff
@@ -45,10 +46,7 @@ export const DeletePage = withCardStateProvider(() => {
         <Text localizationKey={localizationKeys('userProfile.deletePage.actionDescription')} />
 
         <Form.ControlRow elementId={confirmationField.id}>
-          <Form.Control
-            {...confirmationField.props}
-            required
-          />
+          <Form.PlainInput {...confirmationField.props} />
```

Comment:
❓ Same question about `required`.

Reply:
`isRequired` is defined above in the hook, we should be good.